tests/drum/test_args_parser.py

```diff
@@ -149,3 +151,23 @@ def test_enable_strict_validation(self, parser):
             options = parser.parse_args(args)
 
         assert not options.disable_strict_validation
+
+
+class TestBooleanArgumentOptions:
+    def test_boolean_argument_options_passed_through_env_var(self):
+        args = f"{ArgumentsOptions.MAIN_COMMAND} server  --with-error-server --skip-predict".split()
+        with patch.object(sys, "argv", args):
+            unset_drum_supported_env_vars()
+            CMRunnerArgsRegistry.get_arg_parser()
+            os.environ[ArgumentOptionsEnvVars.PRODUCTION] = "True"
+            os.environ[ArgumentOptionsEnvVars.MONITOR] = "False"
+            os.environ[ArgumentOptionsEnvVars.WITH_ERROR_SERVER] = "1"
```

Comment:
how about a test for "0" as well

Reply:
(okay with adding it in a follow-up)

Reply:
Here I test that if  `--with-error-server` is in CMD and in the env var, it ends up with a single  `--with-error-server` in the sys.argv.

The case you referring to, probably is the case with `--skip-predict`

Testing `False` vs `0` doesn't really make sense as its going to be checking that trafaret api works correctly